src/components/ArtistPieChart/ArtistPieChart.tsx

```diff
@@ -0,0 +1,292 @@
+import React, { useEffect, useState } from 'react';
+import getRecommendedArtists from '../../services/getRecommendedArtists';
+import './ArtistPieChart.scss';
+
+const getBrands = (brands : []) : string => brands.map((val : {name : string}) => val.name).join(', ');
+const getAge = (age : {ageGroup : string, matchPercentage : number}) : string => `${age.ageGroup}, ${age.matchPercentage}`;
+const getGender = (gender : {male : number, female : number}) : string => {
+  if (gender.male > gender.female) return `Male, ${gender.male}`;
+  if (gender.female > gender.male) return `Female, ${gender.female}`;
+  return 'Both, 50';
+};
+const getLocations = (locations : []) : string => locations.map((val : {name : string}) => val.name).join(', ');
+const ArtistPieChart = () : JSX.Element => {
```

Comment:
too many properties.. Does not look extendable at all.
Think of modifying a single array of object